projects/packages/jetpack-mu-wpcom/package.json

```diff
@@ -73,6 +76,8 @@
 		"@wordpress/url": "4.16.0",
 		"clsx": "2.1.1",
 		"debug": "4.4.0",
+		"events": "^3.3.0",
+		"i18n-calypso": "7.0.0",
```

Comment:
This is probably not a good idea, since i18n of plugins via GlotPress only looks for the functions from `@wordpress/i18n` and will ignore calls to `i18n-calypso`'s `translate()` function. Can we avoid bringing in `i18n-calypso`?